mailpoet/lib/Config/Populator.php

```diff
@@ -238,14 +238,19 @@ private function createDefaultSettings() {
     }
 
     // set default sender info based on current user
-    $sender = [
-      'name' => $currentUser->display_name, // phpcs:ignore Squiz.NamingConventions.ValidVariableName.MemberNotCamelCaps
-      'address' => $currentUser->user_email, // phpcs:ignore Squiz.NamingConventions.ValidVariableName.MemberNotCamelCaps
+    $defaultSender = [
+      "name" => $currentUser->display_name ?: "", // phpcs:ignore Squiz.NamingConventions.ValidVariableName.MemberNotCamelCaps
```

Comment:
Just a tiny detail FYI. If I'm not mistaken, we prefer using single quotes for plain strings in our codebase. This is not a hard rule because in some cases the double quotes are more useful.